modules/oe_theme_content_tender/templates/oe-theme-content-tender-status.html.twig

```diff
@@ -0,0 +1,9 @@
+{#
+/**
+ * @file
+ * Theme override for the status extra field on tender content type.
```

Comment:
Theme override for the status extra field on Call for tender content type.

Reply:
Updated